pkg/pointerdb/pdbclient/client_test.go

```diff
@@ -279,3 +264,31 @@ func TestDelete(t *testing.T) {
 		}
 	}
 }
+
+func TestApiKeyInjector(t *testing.T) {
+	for _, tt := range []struct {
+		APIKey string
+		err    error
+	}{
+		{"abc123", nil},
+		{"", nil},
+	} {
+		injector := apiKeyInjector(tt.APIKey)
+
+		// mock for method invoker
+		var outputCtx context.Context
+		invoker := func(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn, opts ...grpc.CallOption) error {
+			outputCtx = ctx
+			return nil
+		}
+
+		ctx := context.Background()
+		err := injector(ctx, "/test.method", nil, nil, nil, invoker)
+
+		metadata.FromOutgoingContext(ctx)
```

Comment:
What's the purpose of this call? Are you relying on some side effect?

Reply:
Sorry, I missed that. Needs to be removed